service/test/integration1_test.go

```diff
@@ -945,6 +945,23 @@ func Test1NegativeStackDepthBug(t *testing.T) {
 	})
 }
 
+func Test1Issue573(t *testing.T) {
```

Comment:
integration1_test exists to insure that we do not break backwards compatibility with the old API, things shouldn't be added to it. This test is better suited for `proc/proc_test.go`, translating it should be easy.